assets/l10n/app_fr.arb

```
@@ -332,5 +332,6 @@
   "enter_additional_info_req_id": "Entrez l'identifiant de la demande d'information supplémentaire",
   "additional_info_req_id": "Identifiant de la demande d'information supplémentaire",
   "no_access_to_this_page": "Vous n'avez pas accès à cette page",
-  "scheduled_job_settings": "Paramètres des tâches planifiées"
+  "scheduled_job_settings": "Paramètres des tâches planifiées",
+  "scan": "Scanner"
```

RuboCop (1.81.7)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)